src/Article/presentational/ArticleBody.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import { Button } from "semantic-ui-react";
+import { ArticleDetailApi } from "../../api/ApiProps";
+
+const article = {
+    "content" : "hashtag값에 포함된 요소를 해시태그형식으로 렌더링힌다.타입스크립트로 변수나 함수와 같은 자바스크립트 코드에 타입을 정의할 수 있습니다.타입스크립트의 기본 타입에는 크게 다음 12가지가 있습니다.",
+    "image" : "https://files.slack.com/files-pri/T01GPT3UY9J-F01U4D2D5PW/image.png",
+}
+
+interface Props{
+    
+}
+interface State{
+    count: number;
+}
+
+class CounterButton extends React.Component<Props,State>{
+    constructor(props:number){
+        super(props)
+        this.state={count:0}
+    }
+
+    onIncrement = (): void =>{
+        this.setState(
+            ({count})=>({count:count+1})
+        );
+    }
+
+    onDecrement = (): void =>{
+        this.setState(
+            ({count})=>({count:count-1})
+        );
+    }
+
+    render() {
+    const { onIncrement, onDecrement } = this;
+    return (
+      <div>
+        <button type="button" className="favorites-button" onClick={onIncrement}>
+            <i className="heart outline icon"/>
+        </button>
+      </div>
+    );
+  }
+} 
+
+const ArticleBody = ( dummyData: ArticleDetailApi ) => {
+    const {content, hashtags, favorites, wonders, clips} = {...dummyData}
+    
+    return(
+        <main>
+            <article>
+                <section className = "article-content">
+                    <h6>{article.content}</h6>
+                </section>
+                <section className = "article-image">
+                    <img src={article.image} alt='첨부된 이미지 파일'/>
+                </section>
```

Comment:
굳굳! 너무 좋았어요. 이렇게 `ArticleBody` 는 렌더링**만** 담당하는 친구죠! 

그러니까, `ArticleBody`는 데이터를 받아요. 누군가가(container랍니당), 

"야, 이거 줄테니까 넌 이거 받고 그리기만 해라!"

라고 해서 그 역할을 수행하는 거고, 여기서 누군가가 주는 `이거` 는 `ArticleBody` 가 인자로 받는 `dummyData` 랍니다.
심지어 밑에 destructing도 잘하셨어요! `spread` 를 써서 잘 뿌려줬으니, 그 받은 데이터를 뿌리기만 하면 되겠죠?

```suggestion
const ArticleBody = ( dummyData: ArticleDetailApi ) => {
    const {content, author, hashtags, favorites, wonders, clips} = {...dummyData}
    
    return(
        <main>
            <article>
                <section className = "article-content">
                    <h6>{content}</h6>
                </section>
                <section className = "article-image">
                    <img src={author.image} alt='첨부된 이미지 파일'/>
                </section>
```

이렇게요! 

이렇다면 `ArticleBody` 가 받은 `dummyData` 를 똑바로 쓰는 것으로 볼 수 있을 것 같아요.

위에 5번째 줄에 `article` 로 선언하면 그 정보만!을 렌더링하니까.. 재사용이 심각하게 떨어지는 부분이랍니다 ㅜㅜ